modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor4/ClassicEditor.js

```diff
@@ -41,7 +43,14 @@ const ClassicEditor = forwardRef(
 					className="lfr-editable"
 					config={{
 						toolbar: initialToolbarSet,
-						...editorConfig,
+						...{
+							...editorConfig,
+							...{
+								applicationTitle: label
+									? `${label}, ${helpMessage}`
+									: editorConfig.applicationTitle,
+							},
+						},
```

Comment:
It is typically good practice to put `editorConfig` last, so the values can be overriden in `EditorConfigContributor`s. See my suggested solution in the other comment. Is there a reason not to prevent this capability?

Reply:
There was no reason. I will be following the previous recommendation

Reply:
Hi @markocikos, the reason I'm choosing to destructure the editorConfig first is because the editorConfig [already has](https://github.com/liferay/liferay-portal/blob/141594283481bc0295d088d9501f819159f21aea/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorConfigContributor.java#L69-L72) a property called applicationTitle and I'm looking to override it.